cmd/telegraf/telegraf.go

```diff
@@ -214,40 +159,37 @@ func watchLocalConfig(signals chan os.Signal, fConfig string) {
 	signals <- syscall.SIGHUP
 }
 
-func runAgent(ctx context.Context,
-	inputFilters []string,
-	outputFilters []string,
-) error {
+func (a *Telegraf) gather(ctx context.Context) error {
```

Comment:
Why rename runAgent to gather? It doesn't look like it does anything different and it does more than just gather, like loading config, setting up logging.

Reply:
Good catch, so orginially the struct was called `AgentManager` so I renamed it so it wasn't `agent.runAgent`, but that isn't a problem anymore. I renamed it back to `runAgent` like it was originally.